### ISSUE TITLE
Refactor structure to work similar to Django's model package

### ISSUE THREAD

**Issue by bencleary** [bencleary/django-timescaledb#2]
Refactor structure to work with imports from timescale.db import models rather than direct imports. Currently it requires direct verbose imports to the get to the features, following Djangos model package would make it nicer:

```python
from timescale.db import models as tsdb
```

**Comment by bencleary** [comment]
Looks like in the backends the schema causes a circular import when delcaring the field, testing out a solution by using an inner class on the model and using this to determine if its a hypertable or not.

```python

class Metric(TimescaleModel):

          class Timescale:
                    hypertable = True
                    field = "time"

```

**Comment by daadu** [comment]
>
> ```python
> class Metric(TimescaleModel):
> 
>           class Timescale:
>                     hypertable = True
>                     field = "time"
> ```


Hey @bencleary 

This looks like a good method to store "meta", one issue though - this wont trigger migrations (in backend creating or migrating to "hypertable") - one workaround for that would be to have "special migrations opertaions" (like `migrations.CreateHypertable(name='db_name', field='time', migrate=True)`) written which user have to manually put in migration file.

**Comment by daadu** [comment]
I have thought about it - having `Meta` like class for timescale related info (django fails for any arbitaary info in `Meta`).

I was pondering on how to "declaretively" define - retention/compression/etc policies that Timescale provides, `MyModel.TimescaleMeta` kind of approach wont help - instead I have thought of defining various "migration operations" like `migrations.CreateRetentionPolicy`, etc and user will just put them in migration file.


So to summarise, this approach as I see from current implmentation has only one advantage - that the "partition column", could be any arbitarily named field (currently in README it says to have it named "time", not sure why though? as in queries too, the column name is passed.). Am I right? Do we see any other (or any!) use case?

**Comment by daadu** [comment]
@ schlunsen Why is the name "time" for partion column necessary (as mentioned in README)?
